staging/src/k8s.io/apiserver/pkg/endpoints/groupversion.go

```diff
@@ -76,6 +77,8 @@ type APIGroupVersion struct {
 	// EnableAPIResponseCompression indicates whether API Responses should support compression
 	// if the client requests it via Accept-Encoding
 	EnableAPIResponseCompression bool
+
+	OpenAPIGetter openapi.Getter
```

Comment:
I still think "source" is better than "getter" :)

Also I'm not sure if this makes sense in the group-- I thought the group was constructed before the api installer is called, is that not true?

Reply:
Yes I agree with source, I changed it to source then back to getter when I copied something from kubectl. Would you rather have it in the installer? the only fields currently in installer are a groupversion, prefix, timeout, and a bool enabling APIResponseCompression.

GroupVersion has typers, serializer, convertor, etc